x11/windowmaker/Portfile

```
@@ -18,15 +20,23 @@ long_description \
     It is also free software, with contributions being made \
     by programmers from around the world.
 
-master_sites \
-    http://windowmaker.org/pub/source/release/
+master_sites http://windowmaker.org/pub/source/release/
 
 distname    WindowMaker-${version}
-checksums	rmd160 320f29bb930b046ebdb796dc0d8b2dfdfa551c1b \
-            sha256 b5eb88eccbf9a196a617594717797aa38f5c38d13d2ee40e914e153d09c9975a
+distfiles ${distname}.tar.gz
+checksums	rmd160 3c78acef4e535f4b2230e0e8cf5155c54359a61e \
+    sha256 f22358ff60301670e1e2b502faad0f2da7ff8976632d538f95fe4638e9c6b714 \
+    size 3397167
```

Specifying `distname` is unnecessary here because the value you've set it to is its default.
```suggestion
checksums	rmd160	3c78acef4e535f4b2230e0e8cf5155c54359a61e \
		sha256	f22358ff60301670e1e2b502faad0f2da7ff8976632d538f95fe4638e9c6b714 \
		size	3397167
```